logging/logger.js

```diff
@@ -1,40 +1,12 @@
 ﻿var winston = require('winston');
```

Comment:
Couple thoughts--

- There is still some value in using Winston since it gives you a consistent set of .info, .warn, etc log level methods. It might be worth looking at our boilerplate [logging setup](https://github.com/LeisureLink/boilerplate-api/blob/master/lib/Logger.js). We also use Winston to enforce some standards, like a timestamp, on the output messages even though we're still just logging to the console.
- Another thing to note in our logger is that if a developer really wants to, they can set a local environment variable and get an output file. This variable is never set in our deployment environments though.